website/templates/public/pages/meeting.mako

```diff
@@ -64,6 +58,15 @@
     <script type="text/javascript">
         window.contextVars = window.contextVars || {};
         window.contextVars.meetingData = ${data};
+        
+        % if meeting['active'] and meeting['info_url']:
+            window.contextVars.tbInstructions =  'withLink';
+            window.contextVars.tbInstructionsLink =  '${ meeting['info_url'] }';
```

Comment:
Why have 2 variables here? You could just have `tbInstructionsLink` and check if it is defined in `conference.js`.